### DISCUSSION TITLE
Weird Tabs Bar?

### DISCUSSION THREAD

**Discussion by KoopsF** [sumatrapdfreader/sumatrapdf#4053] [Q&A]
![Screenshot 2024-01-30 191817](https://github.com/sumatrapdfreader/sumatrapdf/assets/11694721/21f465e5-9a63-4308-b172-3e3a07b11e70)

I started SumatraPDF for the first time and I think the Tabs bar is looking weird, or is it me?

**Comment by GitHubRulesOK** [comment]
That's normal for some versions as default is light Title bar (yours is blueish)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c79c1fff-f7cc-41cb-8440-f6932d36b5cc)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/31e5950e-ff8e-4d9c-9bf1-8c047b486025)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2b5678a9-b05f-4a60-8b1d-eca99d512b48)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/1231ecbd-2268-4ab1-a9f4-5e6c135147ce)

**Comment by KoopsF** [comment]
Hmm, strange... An older version seems to have a cleaner Tabs Bar. I've attached a video of that version.

Thanks for the help, though!

https://github.com/sumatrapdfreader/sumatrapdf/assets/11694721/9ccbef10-534a-465a-917c-8fd80ca18d30
